.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement, Impact/Severity: Low, Priority: Low, waiting for collaborator
+  review'
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like.(Which feature would you like to see in the project? A user story is preferred.)**
```

Comment:
i would prefer if the document had questions we can use to create the user story ourselves instead of directly asking for a user story.
what role best describes you?
what feature would you like to see?
why would you want this feature?
which alternatives are you using at the moment?

Reply:
[`e400c3c` (#26)](https://github.com/techlauncher-mlai-edge-physics/physics_in_the_browser/pull/26/commits/e400c3c565e503bd0519615b7519ea1179c4584b)